optical/converter/utils.py

```diff
@@ -55,6 +55,12 @@ def get_annotation_dir(root: Union[str, os.PathLike]):
     return Path(root) / "annotations"
 
 
+def find_job_metadata_key(json_data: Dict):
```

Comment:
I guess in the case of Adjustment jobs, we may have two keys with `metadata` string. Maybe we can ask user input of which key to be considered, if not provided we can use this.

Reply:
We need to change the constructor params of each class if we need user input for job-name.

Reply:
is this one resolved? @satheeshkatipomu @aman0044

Reply:
Yes, it is kind of handling a rare case. For now, we can go with the current approach.